protoc_plugin/test/protos/foo.proto

```diff
@@ -27,3 +27,7 @@ extend Outer {
   optional Inner inner = 3;
   repeated Inner inners = 4;
 }
+
+// EmptyMessage is used to test unknown fields.
+message EmptyMessage {
```

Comment:
There is a predefined empty message already in protoc_plugin/test/protos/google/protobuf/unittest.proto

Reply:
done